js/scripts/zone_loaders/village_zone_loaders.js

```diff
@@ -13,44 +13,191 @@ const loadVillageCanyon = () => {
 
 const loadVillageField = () => {
     const queueAssets = () => {
-        ASSET_MGR.queueDownload(Decoration.DECORATIONS.flowers.MED_RED_FLOWER.SPRITESHEET);
+        ASSET_MGR.queueDownload(Decoration.DECORATIONS.houses.CHAD_HOUSE.SPRITESHEET);
```

Comment:
I don't like that this involves having the same spritesheet many times in the download queue and cache. There should be one path, even if there are (potentially) many constants for the same path string, there should be one entry in the asset manager.